TACC/Core-Portal
# task/FP-1325 - Add notice to Data Depot

Added hardcoded intro message for A2CPS Community data. This will need to be abstracted later for more flexibility (FP-1375)

## Overview: ##
Needed a notice on A2CPS Community data to not store protected health data

## Related Jira tickets: ##

* [FP-1325](https://jira.tacc.utexas.edu/browse/FP-1325)

## Summary of Changes: ##
Replaced the intro message on Community Data to a hardcoded notice that warns users to not place protected health data. The intro message is a first draft and may need to be updated.

## Testing Steps: ##
1. Adjust settings_custom.py to point Community Data to 'a2cps.storage.community'
2. Add `"noPHISystem": 'a2cps.storage.community'` to `_WORKBENCH_SETTINGS`
3.  Navigate to Community Data and check that the intro message says 'Note: this area is not authorized for protected data (i.e. PHI files). Please do not place any confidential/protected data in this space.'
4. Navigate to the other storage systems and make sure that the intro message says 'This page allows you to upload and manage your files.'

## UI Photos:

## Notes: ##
This is a temporary solution, and we need to update this solution to FP-1375 in the new year.
